plugin/my_asset_manager/MyAssetManagerInterface.py

```diff
@@ -61,24 +62,24 @@ def info(self):
         # to acquire the GIL and enter Python.
         return {constants.kInfoKey_EntityReferencesMatchPrefix: self.__reference_prefix}
 
-    def managementPolicy(self, traitSets, context, hostSession):
+    def managementPolicy(self, traitSets, policyAccess, context, hostSession):
         # The management policy defines which traits the manager is
-        # capable of imbuing queried traitSets with. In this case, we
-        # manage locations of assets only, and only in a read, not a
-        # write, context. Note `LocatableContentTrait` is a trait from
-        # the openassetio-mediacreation library, see :
+        # capable of imbuing queried traitSets with. In this case, the
+        # only workflow we support is to resolve the current location of
+        # pre-existing assets. Note `LocatableContentTrait` is a trait
```

Comment:
[Consider][Minor] 
I think this rephrasing makes it slightly more difficult to connect to what's going on, as we don't use the terminology. (ie, read/access)
```suggestion
        # manager allows read access to the locations of assets. 
        # Note `LocatableContentTrait` is a trait
```
"pre-existing" also seems a tad redundant.

Reply:
Yeah I like your suggestion, squashed it in.